test/fixers/test_crystal_format_fixer_callback.vader

```diff
@@ -0,0 +1,33 @@
+Before:
+  Save g:ale_crystal_format_executable
+  Save g:ale_crystal_format_options
+
+  " Use an invalid global executable, so we don't match it.
+  let g:ale_crystal_format_executable = 'xxxinvalid'
+  let g:ale_crystal_format_options = ''
+
+  call ale#test#SetDirectory('/testplugin/test/fixers')
+
+After:
+  Restore
+
+  call ale#test#RestoreDirectory()
+
+Execute(The crystal format callback should return the correct default values):
+
+  AssertEqual
+  \ {
+  \   'command': ale#Escape('xxxinvalid') . ' format -',
```

Comment:
This test fails with:

```
    (1/2) [EXECUTE] (X) {'command': 'xxxinvalid tool format -'} should be equal to {'command': 'xxxinvalid format -'}
```

Make sure the test matches exactly what the `ale#fixers#crystal#Fix()` function returns. Including white spaces..